docs/user/tutorial/3_viz.rst

```diff
@@ -277,8 +284,8 @@ with the plot without recomputing the coordinates.
 .. image:: 3_viz_files/3_viz_13_0.png
 
 
-In this plot, each blue dot represents a simulated election built from
-1000 PL ballots, and each blue dot is likewise 1000 BT ballots, using
+In this plot, each red mark represents a simulated election built from
+1000 PL ballots, and each blue mark is likewise 1000 BT ballots, using
 the same preference interval. The marker, x or o, denotes the preference
 interval type. It’s very important to remember that the x axis and y
 axis numbers do not mean ANYTHING in an MDS plot—there’s literally a
```

Comment:
We should probably remove this sentence since there are no more axis numbers on the plot